charts/kube-prometheus-stack/values.yaml

```diff
@@ -1369,9 +1369,9 @@ prometheusOperator:
     patch:
       enabled: true
       image:
-        repository: jettech/kube-webhook-certgen
-        tag: v1.5.2
-        sha: ""
+        repository: k8s.gcr.io/ingress-nginx/kube-webhook-certgen
+        tag: v1.0
+        sha: "f3b6b39a6062328c095337b4cadcefd1612348fdd5190b1dcbcb9b9e90bd8068"
```

Comment:
Why was the SHA value added here? The chart uses nowhere else sha values for images, only tags.

Reply:
The key was there and it seemed right ™️  to do but now that you mention it it does seems like it would be a breaking change if someone was overriding only the repository and tag... :/

Reply:
Exactly, this was a breaking change for us and we therefore had to override this value inside our values file by setting it to `sha: ""`.

Reply:
Well that's quite unfortunate indeed -- guess I'll wait for a maintainer to chime in as to what's the best course of action here (patch release, keep as-is, change on main...)

Reply:
I guess it was merged with sha?

Reply:
Yep, didn't really hear back from anyone on the topic 🤔